### DISCUSSION TITLE
Alpine 3.19 images now available

### DISCUSSION THREAD

**Discussion by lbussell** [dotnet/dotnet-docker#5091] [Announcements]
# Alpine 3.19 images now available

Alpine 3.19 container images have been released for .NET 6.0, 7.0, and 8.0.

The following Docker repos have been updated:

* dotnet/sdk - [Microsoft Asset Registry](https://mcr.microsoft.com/product/dotnet/sdk/about) | [DockerHub](https://hub.docker.com/_/microsoft-dotnet-sdk)
* dotnet/aspnet - [Microsoft Asset Registry](https://mcr.microsoft.com/product/dotnet/aspnet/about) | [DockerHub](https://hub.docker.com/_/microsoft-dotnet-aspnet)
* dotnet/runtime - [Microsoft Asset Registry](https://mcr.microsoft.com/product/dotnet/runtime/about) | [DockerHub](https://hub.docker.com/_/microsoft-dotnet-runtime)
* dotnet/runtime-deps - [Microsoft Asset Registry](https://mcr.microsoft.com/product/dotnet/runtime-deps/about) | [DockerHub](https://hub.docker.com/_/microsoft-dotnet-runtime-deps)

## Details
You must currently opt-in to Alpine 3.19 by using the version specific tags (e.g. `8.0-alpine3.19`).

In accordance with our tag policy, floating Alpine-based tags for .NET 7.0 and 8.0 (e.g. 8.0-alpine) will continue to reference Alpine 3.18 until the next servicing release on February 13, 2024. At that time the floating tags will be updated to reference Alpine 3.18, with one exception as noted below. This provides an evaluation period for the new version before automatically updating everyone to the new version. Once the floating tags are updated to Alpine 3.18, an announcement will be posted.

PowerShell is currently not supported on Alpine 3.19 on .NET 6 due to https://github.com/PowerShell/PowerShell/issues/20945. We plan to delay moving .NET 6 SDK `alpine` floating tag to 3.19 until that issue is resolved. If your use of the Alpine SDK image doesn't rely on PowerShell, you can opt-in to Alpine 3.19 using the version specific tag (e.g. `6.0-alpine3.19`). Once the issue is resolved, floating tags will be updated to Alpine 3.19 and another announcement will be posted.

**Comment by tvbishan** [comment]
There seems to be something wrong with the Docker repository. An error page is displayed in all of the Docker repos above.

![image](https://github.com/dotnet/dotnet-docker/assets/12027448/87b5ce29-bdb3-4d2c-a7d4-08d557e16c7e)

**Reply by lbussell** [reply]
Thanks for pointing this out. I added links to the [Microsoft Asset Registry](https://mcr.microsoft.com/) so you can still view the tags while we look at this issue. The DockerHub content is also mirrored from the various README files at the root of this repo in case you need to see what was there, but everything should be on MAR.

**Comment by csrakowski** [comment]
Small typo: should `until the next servicing release on February 13, 2023` not say 2024?

**Reply by lbussell** [reply]
Yes, corrected, thanks :)
